lib/matplotlib/backend_bases.py

```diff
@@ -2763,6 +2763,10 @@ def draw_rubberband(self, event, x0, y0, x1, y1):
         """Draw a rectangle rubberband to indicate zoom limits"""
         pass
 
+    def remove_rubberband(self):
```

Comment:
Ah, well then porting this solution to the toolbase framework should be straight forward since it implements the same draw/remove concept.